wp-content/themes/haruoshi_origin/src/scss/archive-works.scss

```
@@ -0,0 +1,29 @@
+@import "variables";
+@import "mixin";
+
+.works {
+	max-width: $contents-max-width;
+	margin: 0 auto;
+	&__article-list {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-between;
+	}
+	&__article-list::after{
+		content:"";
+		display: block;
+		width: 32%;
+	}
```

```suggestion
	&__article-list {
		display: flex;
		flex-wrap: wrap;
		justify-content: space-between;
		&::after{
			content:"";
			display: block;
			width: 32%;
		}
	}
```
書き方としてはこっちのほうがスマート。
ただこの疑似要素はなんのために存在するんだ？

---

workが3の倍数じゃないときに気持ち悪くしないため！